### PR TITLE
Fix Unitary Fund Community Manager ad

### PR DESCRIPTION
**Pull request by nathanshammah**
Fix Unitary Fund Community Manager job ad description from PR #42, as there I made the mistake of copying the wrong job ad.

### CODE REVIEW

File: jobs/2023-08-31_unitaryfund.yaml [hunk 1]
```diff
@@ -1,34 +1,20 @@
 entity: Unitary Fund
 title: Chief of Staff
```
Comment (nathanshammah) [inline jobs/2023-08-31_unitaryfund.yaml]:
```suggestion
title: Community Manager
```

File: jobs/2023-09-01_unitaryfund.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+entity: Unitary Fund
+title: Community Manager
+url: https://unitary.fund/jobs/community_manager/
+percentTime: 100
+percentOSS: 100
+deadline: 2023-11-31
+expires: 2023-11-31
```
Comment (nathanshammah) [inline jobs/2023-09-01_unitaryfund.yaml]:
```suggestion
deadline: 2023-10-31
expires: 2023-11-30
```

**Comment by nathanshammah** [comment]
@stefanv this is ready for review. Thanks in advance.

**Comment by stefanv** [comment]
Thanks! Your post is live on the board, with permalink https://jobs.numfocus.org/job/2023-09-01_unitaryfund

**Pull request merged**